modules/probe.sh

```diff
@@ -218,7 +218,7 @@ probe_initrd_in_boot_and_root() {
   for boot in work/rootfs/boot work/boot; do
     initrd_image_path=$(probe_initrd_image ${boot})
     if [ -n "${initrd_image_path}" ] && [ "${boot}" == "work/boot" ]; then
-      sudo cp ${initrd_image_path} ${target_rootfs_dir}/boot
+      sudo cp ${initrd_image_path} work/rootfs/boot
```

Comment:
It does look correct to me, especially since `target_rootfs_dir` does not occur anywhere else in this file. @mirzak, do you concur?

Reply:
Yes this look good to me. It is a path code that you do not normally enter and for sure not one we enter in our conversion runs during testing, hence went by unnoticed.